src/main/java/tc/oc/pgm/rotation/PGMMapOrderProvider.java

```diff
@@ -0,0 +1,13 @@
+package tc.oc.pgm.rotation;
+
+import tc.oc.pgm.map.PGMMap;
+
+public interface PGMMapOrderProvider {
```

Comment:
Also the name of this interface is a little ambiguous.

Reply:
The point of the name is it should be open to basically whatever kind of ordering that can tell you what map should play next. getNextMap can be null, and pollNextMap must return a map.

Reply:
Makes sense, ok with `PGMMapOrder`

Reply:
I have added `PGMMapOrder`